tests/boot/boot_to_firstboot.pm

```diff
@@ -0,0 +1,29 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary:
```

Comment:
I don't think you need this module, we are using function 'YuiRestClient::connect_to_app_running_system()' for running system instead of the one below, with longer timetout should reach it. You could reuse solution /lib/YaST/Module.pm and skip all except the connection adding some conditional variable, and use it in yast2_firstboot.pm something like `YaST::Module::open`. In that way is not perfect but we agreed that we would improve that part and at least all is in the same place and no code is repeated.

Reply:
I chose to do this way as it makes it easier to work on firstboot modules coming next, as when something goes wrong in the firstboot module, you can quickly see the port to connect to in the previous module and connect with the rest-api server, or see if it failed to connect completely. connect_to_app_running_system does not tell you the port used. So yes, the module can be completely skipped but it really helps with troubleshooting.

Reply:
We should display the info as soon as we got it. It should be displayed before this point, see [comment](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/12108/files#r605518293)

Reply:
in fact, connect_to_app have 500 second of timeout, so should work without waiting for any needle.